.github/workflows/presubmit-c.yml

```diff
@@ -19,7 +19,8 @@ jobs:
     strategy:
       matrix:
         container:
-        - 'debian:latest'
+        - 'debian:stable'
+        - 'debian:testing'
```

Comment:
I think you need to change the if-clause on line 30 to use https://docs.github.com/en/actions/learn-github-actions/expressions#startswith.

Reply:
Whoops, done.